compiler/rustc_infer/src/infer/combine.rs

```diff
@@ -228,6 +228,7 @@ impl<'infcx, 'tcx> InferCtxt<'infcx, 'tcx> {
         ct: &'tcx ty::Const<'tcx>,
         vid_is_expected: bool,
     ) -> RelateResult<'tcx, &'tcx ty::Const<'tcx>> {
+        debug!("unify_const_variable: param_env={:?}", param_env);
```

Comment:
Please use `#[instrument(level=debug, skip(self))]` on the function instead of a `debug!` invocation